e2e/staticpvc.go

```diff
@@ -159,3 +159,116 @@ func validateRBDStaticPV(f *framework.Framework, appPath string, isBlock bool) e
 	execCommandInPod(f, cmd, rookNamespace, &listOpt)
 	return nil
 }
+
+func validateCephFsStaticPV(f *framework.Framework, appPath, scPath string) error {
+	opt := make(map[string]string)
+	var (
+		cephFsVolName = "staticCephFsVol"
+		groupName     = "e2e"
+		fsName        = "myfs"
+		pvName        = "pv-name"
+		pvcName       = "pvc-name"
+		// minikube creates default class in cluster, we need to set dummy
+		// storageclass on PV and PVC to avoid storageclass name missmatch
+		sc = "storage-class"
+	)
+	const secretName = "cephfs-static-pv-sc" // #nosec
+
+	c := f.ClientSet
+
+	listOpt := metav1.ListOptions{
+		LabelSelector: "app=rook-ceph-tools",
+	}
+
+	fsID, e := execCommandInPod(f, "ceph fsid", rookNamespace, &listOpt)
```

Comment:
normally we use err to represent an error, please use err  instead of e

Reply:
err is used below in the same function, have to use a different variable because return type is different.